pages/common/typeinc.md

```diff
@@ -0,0 +1,18 @@
+# typeinc
+
+> Typeinc, an ncurses based terminal typing speed test program, written in python.
```

Comment:
```suggestion
> An ncurses based terminal typing speed test program, written in Python.
```
We avoid mentioning the command name in the description.

Reply:
Okie, anything else? Ill fix everything in 1 commit.

Reply:
> Okie, anything else? Ill fix everything in 1 commit.

I added one more suggestion.